src/Geom_Poly.f90

```diff
@@ -361,20 +373,141 @@ ELEMENTAL FUNCTION isCircle_PolygonType(thisPoly) RESULT(bool)
     ENDFUNCTION isCircle_PolygonType
 !
 !-------------------------------------------------------------------------------
+!> @brief Determines whether a polygon meets the criteria for being an arc
+!>        sector. In short, the polygon must have exactly two quadratic edges
+!>        and exactly two non-quadratic edges. A circle is not an arc sector.
+!> @param thisPoly The polygon type to query
+!> @param bool The logical result of the operation
+!>
+    ELEMENTAL FUNCTION isSector_PolygonType(thisPoly) RESULT(bool)
```

Comment:
Why is this elemental?

Reply:
Other checking routines were `ELEMENTAL`, like `%isCircle`, so I was just sticking with the same convention.